gst/parser/gstttmlparse.cpp

```diff
@@ -55,11 +55,12 @@
 #include <glib.h>
 
 #include "gstttmlparse.h"
-#include "samiparse.h"
-#include "tmplayerparse.h"
-#include "mpl2parse.h"
-#include "qttextparse.h"
-#include "ttmlparse.h"
+//#include "samiparse.h"
```

Comment:
Note that github shows that we renamed the file ("gst/parser/gstttmlparse.c → gst/parser/gstttmlparse.cpp") and so we should rename back too. Do not worry that there will be a mix of C and C++ files. 

Maybe finish with whitespace first, and then create new GST task and just continue from there with C/C++ renames and fitting